.gitignore

```diff
@@ -7,4 +8,4 @@ BiblioPixel.egg*
 *.pyc
 .cache
 .coverage
-htmlcov
\ No newline at end of file
+htmlcov
```

Comment:
Ah, p00p.  Sorry about that, I'll catch that in review next time.  Is there a way to quickly add a Travis rule here?